exec/java-exec/src/main/java/org/apache/drill/exec/store/mock/DateGen.java

```diff
@@ -44,13 +44,16 @@ public class DateGen extends AbstractFieldGen {
   public DateGen() {
     // Start a year ago.
     baseTime = System.currentTimeMillis() - ONE_YEAR;
-    fmt = new SimpleDateFormat("yyyy-mm-DD");
+    fmt = new SimpleDateFormat("yyyy-MM-dd");
   }
 
   @Override
   public void setValue() {
-    long randTime = baseTime + baseTime + rand.nextInt(365) * ONE_DAY;
-    String str = fmt.format(new Date(randTime));
-    colWriter.setString(str);
+    final long randTime = baseTime + baseTime + rand.nextInt(365) * ONE_DAY;
```

Comment:
@jnturton @cgivre this doesn't look right - baseTime is a big number but we add it twice so the result will be a very big number, a long way in the future.